.github/workflows/build.yml

```diff
@@ -124,7 +137,7 @@ jobs:
           DISTRO=ubuntu-$(grep DISTRIB_RELEASE /etc/lsb-release | cut -f 2 -d =)-aarch64
           NAME=llvm-mingw-$TAG-ucrt-$DISTRO
           mv llvm-mingw $NAME
-          tar -Jcf ../$NAME.tar.xz --format=ustar --numeric-owner --owner=0 --group=0 $NAME
+          tar -Jcf ../$NAME.tar.xz --format=ustar --numeric-owner --owner=0 --group=0 --mtime="$BUILD_DATE" $NAME
```

Comment:
This is in a separate step, you'd need to apply the `BUILD_DATE` env on this step as well

Reply:
fixed